phys2bids/cli/run.py

```diff
@@ -134,6 +134,32 @@ def _get_parser():
         "or just one TR if it is consistent throughout the session.",
         default=None,
     )
+    optional.add_argument(
+        "-esttakes",
+        "--estimate_takes",
+        dest="estimate_takes",
+        action="store_true",
+        help="Run automatic algorithm to estimate clusters of triggers, i.e. the "
+        "'takes' or 'runs' of fMRI. Useful when sequences were stopped and restarted, "
+        "or when you don't know how many triggers or trs you have in each take. "
+        "This might work 95%% of the time. Default is False.",
```

Comment:
"This might work 95%% of the time" is a funny sentence! Can we be clearer on why it may not? Or can we say something like "please check this output/plot carefully to see if it has done X task correctly"